tests/util/golinter/rules/no_sleep_rule.go

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Istio Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package rules
+
+import (
+	"go/ast"
+	"go/token"
+)
+
+// NoSleepRule requires that time.Sleep() is not allowed.
+type NoSleepRule struct{}
```

Comment:
Since we already moved these to the Rules package, I'd suggest we just remove the Rule suffix from the type and filename. 

It will make the ID looks more clear as well.

Reply:
Make sense. Done. Thanks!